### PR TITLE
[stable/weave-scope]: Add nodeSelector, tolerations

### PR DESCRIPTION
**Pull request by ydemetriades**
Signed-off-by: Yiannis Demetriades <yiannis.demetriades@gmail.com>

#### Special notes for your reviewer:
Include nodeSelector and tolerations to frontend, agent and cluster-agent to deploy on specific nodes

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/weave-scope/charts/weave-scope-agent/templates/daemonset.yaml [hunk 1]
```diff
@@ -63,6 +63,14 @@ spec:
               mountPath: /var/run/scope/plugins
             - name: sys-kernel-debug
               mountPath: /sys/kernel/debug
+      {{- if .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml .Values.nodeSelector | nindent 8 -}}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations:
```
Comment (bboreham) [inline stable/weave-scope/charts/weave-scope-agent/templates/daemonset.yaml]: Is it ok to have two `tolerations:` sections?  (there's another one on line 25)
(I know nothing about helm)

Comment (ydemetriades) [reply stable/weave-scope/charts/weave-scope-agent/templates/daemonset.yaml]: Indeed. Nice catch @bboreham Missed that daemonset had already a tolerations section. Refactoring now...

Comment (ydemetriades) [reply stable/weave-scope/charts/weave-scope-agent/templates/daemonset.yaml]: @bboreham fixed daemonset.yaml __tolerations__ section.
Please review it again!

**Pull request closed without merging**